package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "marsappapi",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "@types/express": "^4.17.12",
```

Comment:
Types should always be added as development dependencies (devDependencies, with -D flag)